test/Common/Button.test.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import Button from '../../src/components/Common/Button';
+
+describe('Check Component creation', () => {
+    const element = shallow(<Button type='submit' label='Submit' className='b1'/>);
+    test('Button component', () => {
+        expect(element).toMatchSnapshot();
+        expect(element.text()).toEqual('Submit');
+    });
+    test('Button props', () => {
+        expect(element.prop('type')).toEqual('submit');
+        expect(element.prop('className')).toEqual('b1 btn');
+    });
+    test('Button click 1', () => {

```

Comment:
This test name is accurate? Because you're testing the disabled prop, not click. Also for property tests, I believe snapshot tests are better option. But when you're testing actions you could test if the action is called by jest. (Apply same for all other tests please)